test/introspection/ERC165Checker.test.js

```diff
@@ -0,0 +1,151 @@
+const ERC165Checker = artifacts.require('ERC165CheckerMock');
+const ERC165NotSupported = artifacts.require('ERC165NotSupported');
+const ERC165InterfacesSupported = artifacts.require('ERC165InterfacesSupported');
```

Comment:
Travis is erroring at this line, because this contract doesn't exist.

Reply:
ahh, git didn't track me renaming this file and I forgot to add it.